### ISSUE TITLE
plaidml.exceptions.PlaidMLError: Could not find PlaidML configuration file: "experimental.json"

### ISSUE THREAD

**Issue by mrichman** [plaidml/plaidml#969]
Tried `pip install plaidml-keras plaidbench` using both virtualenv and with `--user`. Same error both times.

macOS 10.15.3
python 3.7.7
pip 20.0.2

**Comment by limadm** [comment]
#370 has some indication on how to fix this issue.

**Comment by nandkishore1** [comment]
Did you solve the issue?
If yes 
Plz share it

**Comment by limadm** [comment]
> Did you solve the issue? If yes Plz share it

Yes, you just need to reinstall using pip3 in user-mode. Please follow the thread on #370
